packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -3480,6 +3484,14 @@ class InputDecoration {
   /// a default height based on text size.
   final BoxConstraints? constraints;
 
+  /// Optional widget that appears below the [InputDecorator.child] and the border.
+  /// If non-null, the border's color animates to red and the [helperText] is
+  /// not shown.
+  ///
+  /// If both [errorText] and [error] are specified, then [error]
+  /// is shown.
+  final Widget? error;
```

Comment:
Is there a use case for specifying both `errorText` and `error`? If not, it'd probably be better to disallow specifying both.

Ah, in fact it looks like the pairs label/labelText, prefix/prefixText, suffix/suffixText all allow at most one to be specified in each pair. So this should probably do the same thing.